modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache

```diff
@@ -127,7 +127,20 @@ export const {{classname}}AxiosParamCreator = function (configuration?: Configur
                     {{paramName}};
                 {{/isDate}}
                 {{^isDate}}
+                {{#isExplode}}
+                {{#isPrimitiveType}}
                 localVarQueryParameter['{{baseName}}'] = {{paramName}};
+                {{/isPrimitiveType}}
+                {{^isPrimitiveType}}
+                let param: keyof typeof {{baseName}};
```

Comment:
```suggestion
                let param: keyof typeof {{baseName}};
```

please also re-generate the samples

Reply:
No any changes after re-generating samples.
What I've done is, (under current commit)
```bash
mvn clean package
./bin/generate-samples.sh ./bin/configs/typescript-axios*
# No any changes after execute generate samples command lines.
```
Did I do something wrong?

Reply:
ok then probably there were no configs for this use case, the CI is green, so all good.